### PR TITLE
New FMS councils page

### PR DESCRIPTION
**Pull request by wrightmartin**
fixmystreet professional fixmystreet

### CODE REVIEW

**Comment by wrightmartin** [comment]
- Needs development work as the enquiry form has changed (removed most fields, added mailing list opt-in)

File: templates/web/fixmystreet.com/about/council.html [hunk 1]
```diff
@@ -4,66 +4,102 @@
...
+          <h3>Unitary and two-tier authorities</h3>
+          <p>Handle all reports or a subset of reports by category, location or a combination of both.</p>
         </div>
         <div class="councils-features__feature">
-          <div class="councils-features__thumb councils-features__thumb--process"></div>
-          <h3>Streamline your process </h3>
-          <p>FixMyStreet is the first case-management service designed with councils, for councils. It streamlines your processes right through from customer service, to inspection, instruction and resolution.</p>
+          <div class="councils-features__thumb councils-features__thumb--syncronised"></div>
+          <h3>Synchronised with FixMyStreet.com</h3>
+          <p>Handle all reports or a subset of reports by category, location or a combination of both.</p>
```
Comment (abimysoc) [inline templates/web/fixmystreet.com/about/council.html]: The text here is the same as for the previous feature, which I'm guessing is not intentional?

**Comment by dracos** [comment]
"Needs development work as the enquiry form has changed (removed most fields, added mailing list opt-in)" - I designed the form to not require development input no matter what fields it had :)

It needs (hidden or not) name, email, subject, message; then it automatically adds anything with a name starting `extra.` (look at the ones you're removing to see what I mean). So you'll want to call it `extra.maillinglist` or something and it should then work.

**Comment by zarino** [comment]
@wrightmartin I’ve optimised the SVGs, and switched out the houdini show/hide thing for a jQuery version, which bypasses the odd webkit border-top/right issue we were facing.

The horizontal scroll on narrow screens is a bit yucky. I’ve saved a bit of time on other FMS tickets this sprint, so I’ll take a quick look tomorrow at how we could rearrange the table to a more vertical-friendly format on narrow screens.

**Comment by zarino** [comment]
Pricing table now works nicely on narrow screens:

screen shot 2017-05-19 at 15 34 28

But, while testing that out, I’ve noticed the flexbox stuff further up the page doesn't seem to be working on iOS. Needs investigating.

screen shot 2017-05-19 at 15 36 39

**Comment by zarino** [comment]
@martin @jukesie I’ve fixed the outstanding issues on this and squashed the commits. Should be fine for us to merge and deploy on Monday.

**Comment by dracos** [comment]
Please have a commit message starting `[fixmystreet.com] ` to match the others, thanks!

**Comment by zarino** [comment]
@dracos Done!

**Pull request merged**